files/ru/learn/css/css_layout/index.md

```diff
@@ -41,7 +41,7 @@ slug: Learn/CSS/CSS_layout
   - : **CSS Media Query** позволяют писать CSS под разные условия, например "область просмотра шире 480 пикселей". Медиавыражения являются ключевой частью отзывчивого веб-дизайна, поскольку они позволяют создавать различные макеты в зависимости от размера области просмотра. Они также могут использоваться для определения других особенностей среды, в которой работает ваш сайт, например, использует ли пользователь сенсорный экран, а не мышь. В этом руководстве вы сперва познакомитесь с синтаксисом медиавыражений, а после разберёте их использование на интерактивном примере, показывающем, как простой дизайн можно сделать отзывчивым.
 - [Устаревшие методы вёрстки](/ru/docs/Learn/CSS/CSS_layout/Legacy_Layout_Methods)
   - : Grid-системы - это очень распространённая возможность, используемая в CSS layouts, и до CSS Grid Layout они, как правило, реализовывались с помощью floats или других возможностей вёрстки. Вы представляете свою вёрстку в виде заданного числа столбцов (например, 4 или 6), а затем вы помещаете содержимое в эти воображаемые столбцы. В этом разделе мы рассмотрим, как работают эти старые методы, чтобы вы понимали, как они использовались, если столкнётесь со старыми проектами.
-- [Поддержка старыми браузерами](/en-US/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers)
+- [Поддержка старыми браузерами](/ru/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers)
```

Comment:
@yin1999 thanks for helping. Could you help me to understand the new rules? Now we should place link only for our "own locales"? In that case, it's ok that the page https://developer.mozilla.org/ru/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers isn't existing?

Reply:
Yes, yari would automatically falls back to en-US.

![image](https://github.com/mdn/translated-content/assets/15844309/d4683844-d730-4567-9ac8-f239c424c26b)

And doing so can reduce our workload of maintaining documents in the future, without having to update links to other documents after localizing non-existing pages.

For the flaws showing in the preview pages, we have a discuession on it: https://github.com/orgs/mdn/discussions/144.